package.json

```diff
@@ -19,6 +20,7 @@
     "start": "react-scripts start",
     "build": "yarn prettier-check && react-scripts build && yarn eclint && yarn test",
     "test": "react-scripts test --watchAll=false",
+    "test:debug": "react-scripts --inspect-brk test --runInBand",
```

Comment:
Thanks @fcoronelmakingsense I will read the page. In general, I prefer a link than a screenshot I can open in another tab, read the context etc.

In this case, you could generate the link easily in this way:

![image](https://user-images.githubusercontent.com/1157864/52004951-38a53400-24a7-11e9-913d-d548bdce1360.png)

Also, the image and links are fine, but maybe there is a short and clear answer about why did you include this change, it could be useful and time-saver.

Reply:
![image](https://user-images.githubusercontent.com/1157864/52005237-f9c3ae00-24a7-11e9-837e-4e35d054fb78.png)

Wow! it is very impresive! do you have a screenshot about how does it work in your environment?

Reply:
Yes sorry, i use Sublime Text. I can migrate to visual studio code if is better

Reply:
Jaja, me parece que nos estamos entendiendo mal Fede, mi inglés es muy malo. Te llamo y te consulto directamente.

Reply:
Haha, I have a wrong section in my screenshot. We already talked about it.

Let me know when you get Chrome debugging working, it is very interesting. Also, share with me a screenshot, please.

Reply:
I remove for now. need more testing

Reply:
> Mmm, [Facebook](https://facebook.github.io/create-react-app/docs/getting-started) people are disappointing me. A lot of wonderful and magic things that they said in _create-react-app_ articles are not working.

The same with this :(